chain/chain/src/resharding/manager.rs

```diff
@@ -213,11 +222,24 @@ impl ReshardingManager {
             };
             let mem_changes = trie_changes.mem_trie_changes.as_ref().unwrap();
             let new_state_root = mem_tries.apply_memtrie_changes(block_height, mem_changes);
+            drop(mem_tries);
+
             // TODO(resharding): set all fields of `ChunkExtra`. Consider stronger
             // typing. Clarify where it should happen when `State` and
             // `FlatState` update is implemented.
             let mut child_chunk_extra = ChunkExtra::clone(&chunk_extra);
             *child_chunk_extra.state_root_mut() = new_state_root;
+            // TODO(resharding) - Implement proper congestion info for
+            // resharding. The current implementation is very expensive.
```

Comment:
> While we're waiting for the bandwidth scheduler to give us an efficient way to do the same

`ReceiptGroupsQueue` is already fully functional and merged into `master`, I didn't plan to add anything else. What sort of functionality are you looking for?

Reply:
Ah nice! I think all I need is the total size of the buffered receipts. I'll check and come back.